### ISSUE TITLE
Documentation and planning on WCAG 2.1 AA compliance

### ISSUE THREAD

**Issue by taylorsperry** [sourcegraph/sourcegraph#31200]
## Problem to solve

Today, the Sourcegraph web application does not comply with current WCAG guidelines. 

Ensuring our web application becomes--and stays--compliant is essential not just because we have current contractual obligations to do so, but also because:
1. We believe in the value of making our application compliant in furthering our mission
2. Compliance will likely be a requirement of future clients
3. Compliance can be a unique selling point

We want to ensure our application is WCAG 2.1 AA compliant by the end of FY23 (January 2023).

## Measure of success

Any team is able to accurately audit their own areas of ownership and set up automated accessibility testing.

## Solution summary

The Q1 part of this effort is to lay groundwork, share documentation, and line up Accessibility Champions for impacted teams. For Q2 efforts, see the [Auditing](https://github.com/sourcegraph/sourcegraph/issues/31475) and [Fixing](https://github.com/sourcegraph/sourcegraph/issues/31476) roadmap issues. 

Artifacts:
- [Sourcegraph Accessibility Remediation Plan](https://docs.google.com/document/d/1kVinUrMukEEFfqcHS1o36ASAezHKDkSjrBiILvGuDiE/edit)
- [RFC 557: WCAG 2.1 AA Compliance](https://docs.google.com/document/d/1r9W3ZvSaT_CPzzLpp0cZgcFR47Qk8pbT76ATQPiIDRk/edit)
- [Accessibility Champions](https://docs.google.com/spreadsheets/d/1j0c77t5T9hlBno_htYj2Wd3RQzE02vq70l7gQvd4DHc/edit#gid=0)
- [WCAG 2.1 AA Accessibility Project Board](https://github.com/orgs/sourcegraph/projects/238/views/1)
- [How to conduct an accessibility audit](https://docs.sourcegraph.com/dev/background-information/web/accessibility)
- [#wg-accessibility-champions](https://sourcegraph.slack.com/archives/C038K3PR2M7/p1647976803007339)

## What specific customers are we iterating on the problem and solution with?

- https://github.com/sourcegraph/accounts/issues/278

## Impact on use cases

This work contributes to the overall effort to scale our business. 

## Delivery plan

- [x] Step 1: Documentation and planning (February, March, April) 
   - [x] @umpox Create [WCAG 2.1 AA tracking issue](https://github.com/sourcegraph/sourcegraph/issues/31235)
   - [x] Feb 18 @taylorsperry: Identify all relevant teams that will need to be involved in this work
   - [x] March 18 @umpox Add [documentation](https://docs.sourcegraph.com/dev/background-information/web/accessibility) for conducting accessibility audits
   - [x] March 22 @taylorsperry: Form a working group of "accessibility champions." Communications can be directed to the #wg-accessibility-champions channel or @accessibility-champions handle. 
   - [x] March 29 @taylorsperry: Form a comprehensive list of user journeys that need to be audited
   - [x] March 29 @umpox TPGi automated scanning setup
   - [x] April 12 @umpox TPGi automated branch CI scanning
